src/flutter/flutter-wizard.ts

```diff
@@ -0,0 +1,119 @@
+import { WizardOptions } from '../utils/types';
+import * as Sentry from '@sentry/node';
+import * as codetools from './code-tools';
+import * as fs from 'fs';
+
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import * as clack from '@clack/prompts';
+import chalk from 'chalk';
+
+import {
+  confirmContinueIfNoOrDirtyGitRepo,
+  getOrAskForProjectData,
+  printWelcome,
+} from '../utils/clack-utils';
+
+import { traceStep, withTelemetry } from '../telemetry';
+import { findFile } from './code-tools';
+
+export async function runFlutterWizard(options: WizardOptions): Promise<void> {
+  return withTelemetry(
+    {
+      enabled: options.telemetryEnabled,
+      integration: 'android',
+      wizardOptions: options,
+    },
+    () => runFlutterWizardWithTelemetry(options),
+  );
+}
+
+async function runFlutterWizardWithTelemetry(
+  options: WizardOptions,
+): Promise<void> {
+  printWelcome({
+    wizardName: 'Sentry Flutter Wizard',
+    promoCode: options.promoCode,
+  });
+
+  await confirmContinueIfNoOrDirtyGitRepo();
+
+  const { selectedProject, selfHosted, sentryUrl, authToken } =
+    await getOrAskForProjectData(options, 'flutter');
+
+  const projectDir = process.cwd();
+  const pubspecFile = findFile(projectDir, 'pubspec.yaml');
+
+  // ======== STEP 1. Add sentry_flutter and sentry_dart_plugin to pubspec.yaml ============
+  clack.log.step(
+    `Adding ${chalk.bold('Sentry')} to your apps ${chalk.cyan(
+      'pubspec.yaml',
+    )} file.`,
+  );
+  const pubspecPatched = await traceStep('Patch pubspec.yaml', () =>
+    codetools.patchPubspec(
+      pubspecFile,
+      selectedProject.slug,
+      selectedProject.organization.slug,
+    ),
+  );
+  if (!pubspecPatched) {
+    clack.log.warn(
+      "Could not add Sentry to your apps pubspec.yaml file. You'll have to add it manually.\nPlease follow the instructions at https://docs.sentry.io/platforms/flutter/#install",
```

Comment:
Let me know if it's out of scope, but the other wizards show code pastable code snippets so users don't have to leave the wizard and look online for the code.

Example from RN: https://github.com/getsentry/sentry-wizard/blob/fe849febbedb56dc803f79c79d2347b75900c032/src/react-native/javascript.ts#L31-L35